src/main.coffee

```diff
@@ -5,7 +5,15 @@ feedback = require './feedback'
 
 time = -> (new Date()).getTime()
 
-zxcvbn = (password, user_inputs = []) ->
+zxcvbn = (password, options = {}) ->
+  if options instanceof Array
+    user_inputs = options # backward-compatibility
+  else if typeof options === 'object'
+    { user_inputs, feedback_messages } = options
+  else
+    user_inputs = []
+    feedback_messages = {}
```

Comment:
`options` are a bit better because they make future changes easier. I get error when compile, so I suggest the following version:

```
zxcvbn = (password, options = {}) ->
  user_inputs = []
  feedback_messages = {}
  if options instanceof Array
    user_inputs = options # backward-compatibility
  else
    user_inputs = options.user_inputs if options.user_inputs
    feedback_messages = options.feedback_messages if options.feedback_messages
```